buildSrc/src/main/java/Dependencies.kt

```diff
@@ -81,9 +77,9 @@ object Deps {
     const val osslicenses_plugin = "com.google.android.gms:oss-licenses-plugin:${Versions.osslicenses_plugin}"
     const val osslicenses_library = "com.google.android.gms:play-services-oss-licenses:${Versions.osslicenses_library}"
 
-    const val rxKotlin = "io.reactivex.rxjava2:rxkotlin:${Versions.rxKotlin}"
-    const val rxAndroid = "io.reactivex.rxjava2:rxandroid:${Versions.rxAndroid}"
-    const val rxBindings = "com.jakewharton.rxbinding3:rxbinding:${Versions.rxBindings}"
+    const val anko_commons = "org.jetbrains.anko:anko-commons:${Versions.anko}"
```

Comment:
Anko is losing support soon. We shouldn't add it back. Jetpack Compose and the KTX libraries should give us most of what it did.